administration/src/cards/CsvFactory.ts

```diff
@@ -0,0 +1,67 @@
+import { QrCode } from '../generated/card_pb'
+import { convertProtobufToHexCode } from '../util/qrcode'
+import CardBlueprint from './CardBlueprint'
+import { CreateCardsResult } from './createCards'
+import AddressExtensions from './extensions/AddressFieldExtensions'
+import BirthdayExtension from './extensions/BirthdayExtension'
+import NuernbergPassIdExtension from './extensions/NuernbergPassIdExtension'
+import StartDayExtension from './extensions/StartDayExtension'
+import { findExtension } from './extensions/extensions'
+
+export class CsvError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'CSVError'
+  }
+}
+
+export async function generateCsv(codes: CreateCardsResult[], cardBlueprints: CardBlueprint[]) {
+  try {
+    let csvContent =
+      'Name,AddressLine1,AddressLine2,AddressLocation,PassId,Birthday,StartDate,ExpirationDate,CardHash,ActivationCode,StaticUserCode\n'
```

Comment:
I have some additional suggestions. I think we should create a csvExportConfig in nuernberg/config and define there structure of the headline and a fkt that returns the content of a csv content line. Since this is very project specific and should be configurable differently for other upcoming projects if needed

Reply:
Done